spyder/widgets/panels/line_number.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please add the following header

```python
# -*- coding: utf-8 -*-
#
# Copyright © Spyder Project Contributors
# Licensed under the terms of the MIT License
# (see spyder/__init__.py for details)
```

This header needs to be present in all our files :-)